src/Microsoft.Health.Dicom.SqlServer/Features/ExtendedQueryTag/Error/SqlExtendedQueryTagErrorStoreV4.cs

```diff
@@ -81,20 +81,20 @@ public override async Task<IReadOnlyList<ExtendedQueryTagError>> GetExtendedQuer
             using SqlConnectionWrapper sqlConnectionWrapper = await ConnectionWrapperFactory.ObtainSqlConnectionWrapperAsync(cancellationToken);
             using SqlCommandWrapper sqlCommandWrapper = sqlConnectionWrapper.CreateSqlCommand();
 
-            VLatest.GetExtendedQueryTagErrors.PopulateCommand(sqlCommandWrapper, tagPath, limit, offset);
+            V4.GetExtendedQueryTagErrors.PopulateCommand(sqlCommandWrapper, tagPath, limit, offset);
 
             try
             {
                 using SqlDataReader reader = await sqlCommandWrapper.ExecuteReaderAsync(CommandBehavior.SequentialAccess, cancellationToken);
                 while (await reader.ReadAsync(cancellationToken))
                 {
                     (int tagkey, short errorCode, DateTime createdTime, string studyInstanceUid, string seriesInstanceUid, string sopInstanceUid) = reader.ReadRow(
-                        VLatest.ExtendedQueryTagError.TagKey,
-                        VLatest.ExtendedQueryTagError.ErrorCode,
-                        VLatest.ExtendedQueryTagError.CreatedTime,
-                        VLatest.Instance.StudyInstanceUid,
-                        VLatest.Instance.SeriesInstanceUid,
-                        VLatest.Instance.SopInstanceUid);
+                        V4.ExtendedQueryTagError.TagKey,
```

Comment:
Why did ExtendedQueryTagErrorStore change?

Reply:
Thanks for pointing out. There is no need to make these changes. I made them initially just to test things out and forgot to revert. I will update this and other similar changes in the next iteration.

Reply:
Updated code in the latest iteration.